examples/expanding_beam/analysis_expanding_envelope.py

```diff
@@ -104,9 +104,9 @@ def read_time_series(file_pattern):
 assert np.allclose(
     [sig_xf, sig_yf, sig_tf],
     [
-        9.029112e-04,
-        9.029112e-04,
-        1.841402e-06,
+        8.94427191e-04,
```

Comment:
Here the targets were tuned to analytically-determined values, instead of being set to coincide with a low-resolution run.